lms/djangoapps/grades/admin.py

```diff
@@ -0,0 +1,27 @@
+"""
```

Comment:
That's fair. admin.py has to be at the top level but the rest can remain in a sub-directory. I'm going to name is `config` since naming it `admin` causes Django conflicts.